src/rotten-tomatoes/index.js

```diff
@@ -1,3 +1,51 @@
-function rottenTomatoes(grid) {}
+function rottenTomatoes(grid) {
+  const numRows = grid.length;
+  const numCols = grid[0].length;
+  const allDirections = [
+    [-1, 0],
+    [0, 1],
+    [1, 0],
+    [0, -1]
+  ];
```

Comment:
Where did you get this from? Why do you need it?

Reply:
I am using it to check all four directions of each rotten tomatoes for fresh tomatoes.